source/guides/token-management/index.markdown

```diff
@@ -294,6 +309,13 @@ $result = (new \Stormpath\Oauth\VerifyAccessToken($application))->verify();
 
 // Return Type: Stormpath\Resource\AccessToken
 ------
+JwtAuthenticationRequest authRequest = Oauth2Requests.JWT_AUTHENTICATION_REQUEST.builder()
+  .setJwt(oauthGrantAuthenticationResult.getAccessTokenString())
+  .build();
+JwtAuthenticationResult authResult = Authenticators.JWT_AUTHENTICATOR
+  .forApplication(application)
+  .authenticate(authRequest);
+------
```

Comment:
In line 345 you can add Java code to retrieve an `AccessToken` as well. I am not sure why the other SDKs did not add that, but we do support that so we should show the Java Code for that.